packages/warehouses/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "@lightdash/warehouses",
+    "version": "0.96.0",
+    "license": "MIT",
+    "private": true,
+    "main": "dist/index",
+    "dependencies": {
+        "@google-cloud/bigquery": "^5.9.1",
+        "snowflake-sdk": "^1.6.4",
+        "pg": "^8.7.1",
+        "odbc": "^2.4.1",
+        "common": "^0.96.0",
+        "winston": "^3.3.3"
+    },
+    "description": "Warehouse connectors for Lightdash",
+    "devDependencies": {
+        "@types/node": "^14.14.35",
+        "@types/jest": "^27.0.3",
+        "jest": "^27.4.7",
+        "@typescript-eslint/eslint-plugin": "^5.22.0",
+        "@typescript-eslint/parser": "^5.22.0",
+        "eslint": "^8.7.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "ts-jest": "^27.0.3",
+        "eslint-config-airbnb-typescript": "^16.1.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "^2.25.4",
+        "eslint-plugin-json": "^3.0.0",
+        "prettier": "^2.3.2",
+        "prettier-plugin-organize-imports": "^2.3.4",
+        "ts-node": "^9.1.1",
+        "typescript": "^4.2.3"
+    },
+    "scripts": {
+        "lint": "eslint ./src",
+        "format": "prettier --check --ignore-unknown ./src",
+        "test": "jest",
+        "copy-files": "copyfiles -u 1 src/services/warehouseClients/amazon-trust-ca-bundle.crt dist/",
```

Comment:
on `yarn dev` we get the error and the server crashes

Error: ENOENT: no such file or directory, open '/usr/app/packages/warehouses/dist/warehouseClients/amazon-trust-ca-bundle.crt'